movie-showcase/src/routes/managing/+page.svelte

```diff
@@ -0,0 +1,124 @@
+<script lang="ts">
+    import { movieArray } from "../../stores";
+    import { slideDuration } from "../../stores";
+    import { carouselItemCount } from "../../stores";
+
+    var Title: string;
+    let Year: string;
```

Comment:
```suggestion
    let title: string;
    let year: string;
```

The convention is to use `camelCase` for variable names.

Also, the `var` should be replaced by `const` or `let`.

[Read more](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/)